test/SoftMax-CrossEntropyTest.jl

```diff
@@ -0,0 +1,82 @@
+include("../src/layers/LayerBase.jl")
+include("../src/layers/SoftMaxCrossEntropy.jl")
+include("../src/layers/SoftMax.jl")
+include("../src/layers/CrossEntropyLoss.jl")
```

Comment:
Resolved by creating "CrossEntropyTest.jl"; keeping "SoftMax-CrossEntropyTest.jl" just in case.